### DISCUSSION TITLE
Add support for NATS server

### DISCUSSION THREAD

**Discussion by kane50613** [coollabsio/coolify#3852] [Service Template Requests]
Hope we could have one click [nats](https://hub.docker.com/_/nats) service support!

**Comment by abdallah-shaltout** [comment]
Hi!

We would also love to have **NATS server** available as a one-click service in Coolify.

NATS is becoming an essential part of modern microservices and event-driven architectures, and having it as a built-in service template would make deployments much smoother — especially for production environments relying on Coolify.

It would be great to see this added in future releases.

Thanks for the great work you’re doing on Coolify!
